arches_querysets/utils/datatype_transforms.py

```diff
@@ -104,6 +104,7 @@ def concept_list_to_json(self, tile, node):
 def file_list_transform_value_for_tile(self, value, *, languages, **kwargs):
     if not value:
         return value
+    # TODO: this should probably delegate more to self.transform_value_for_tile()
```

Comment:
When we need to upload files themselves we should start looking into calling `self.transform_value_for_tile()` to handle all the file processing. Maybe @aarongundel you can share your findings about what needs adjusting?

Reply:
This works great for the viewer.  I'll open more issues for the editor.